MVVMIOS/Models/Response.swift

```diff
@@ -7,6 +7,44 @@
 
 import Foundation
 
-struct Response: Codable {
+struct Response: Codable, Equatable {
     let name: String
 }
+
+struct News<T: Codable>: Codable {
+    let data: T
+}
+
+struct Pagination: Codable {
+
+}
+
+struct NewsData: Codable {
+    let id: UUID = UUID()
+    let author: String?
+    let title: String?
+    let description: String?
+    let url: String?
+    let source: String?
+    let category: String?
+    let language: String?
+    let country: String?
+
+    enum CodingKeys: String, CodingKey {
+        case id
+        case author, title, description, url, source, category, language, country
+    }
+
+    static var placeholder: [NewsData] {
+        return [
+            .init(author: "test_author", title: "test_title",
+                  description: "test_description", url: "test_url",
+                  source: "test_source", category: "test_category",
+                  language: "test_lan", country: "test_country"),
+            .init(author: "test_author", title: "test_title",
+                  description: "test_description", url: "test_url",
+                  source: "test_source", category: "test_category",
+                  language: "test_lan", country: "test_country")
+        ]
+    }
```

Comment:
Looks like this is only needed for tests right? If so, better to move it into a test file itself, since tests aren't shipped with Production app releases and there's no reason for this code to be in a production release.

Reply:
What can be the best candidate to put this placeholder. Lets say StubNewsData struct and put this placeholder inside the StubObject?

Reply:
You could just move this `static var placeholder...` to an extension on this type, but defined in a test file that is only included in the tests target, therefore not shipped in production.